LayoutTests/imported/w3c/web-platform-tests/pointerevents/pointerevent_hit_test_scroll_visible_descendant.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!DOCTYPE html> <!-- webkit-test-runner [ AsyncOverflowScrollingEnabled=true ] -->
```

Comment:
Please undo this change.

Reply:
No, we need this to run EWS. It will be gone when we rebased after https://github.com/WebKit/WebKit/pull/19664 is merged